openwisp_notifications/handlers.py

```diff
@@ -198,24 +196,20 @@ def send_email_notification(sender, instance, created, **kwargs):
     else:
         target_url = None
     if target_url:
-        description += '\n\nFor more information see {0}.'.format(target_url)
-    mail = EmailMultiAlternatives(
-        subject=subject,
-        body=strip_tags(description),
-        from_email=settings.DEFAULT_FROM_EMAIL,
-        to=[instance.recipient.email],
+        description += _('\n\nFor more information see %(target_url)s.') % {
+            'target_url': target_url
+        }
```

Comment:
This message was added only in text mode while here it seems it's being always added.
I think we need to change `send_email` to have an HTML body and a text body, so we can pass both.